app.py

```diff
@@ -23,12 +47,45 @@ def gamemode():
     """ Select gamemode page."""
     return render_template('gamemode.html')
 
-@app.route('/rank')
-def rank():
-    """ View Score Board page."""
-    return render_template('rank.html')
-
-
+@app.route("/rank", methods=['post', 'get'])
+def rank():  
+    cursor.execute("SELECT * FROM userinfo WHERE game_mode='warmup' ORDER BY score ASC LIMIT 10;")
+    result = cursor.fetchall()
+    return render_template("rank.html", data=result)
+
+@app.route("/getselectinfo", methods=['post','get'])
+def getselectinfo():
+    mode = request.form["game_mode"]
+    body = request.form["game_body"]
+    level = request.form["game_level"]
+    print(mode,body,level)
+    cursor.execute("SELECT * FROM userinfo WHERE userinfo.game_mode='{}' AND userinfo.game_body='{}' AND userinfo.game_level={} ORDER BY score ASC LIMIT 10;".format(mode,body,level))
```

Comment:
Rewrite the SQL operations like [this](https://stackoverflow.com/questions/61036922/how-do-i-prevent-sql-injections-in-flask-sqlalchemy-is-there-a-better-way-to-lo) to avoid [SQL Injection](https://en.wikipedia.org/wiki/SQL_injection).

Reply:
Add a `FIXME` comment here if you don't want to fix the problem this time.